relationships.md

```diff
@@ -104,6 +106,11 @@ A sample CURL command would be:
 curl -i -X POST 'https://dspace7-entities.atmire.com/rest/api/core/relationships?relationshipType=1' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
 ```
 
+Including a name variant would result in:
+```
+curl -i -X POST 'https://dspace7-entities.atmire.com/rest/api/core/relationships?relationshipType=1&leftWardLabel=Name%20variant%201' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
```

Comment:
I think we may want to improve this example or explain it further.  I'm assuming what you mean is that name variants are always specified on labels?  So, if your name variant is "Smith, Jane" then you'd assign the value of the appropriate label to be "Smith, Jane".  Is that correct?

If that's correct, I think we need to be clearer in our contract about the fact that you are expected to use the appropriate label to assign a name variant (and that in this example, the name variant assigned is "Name variant 1").  Currently, this example is vague as it's unclear if "Name variant 1" is the name variant text itself...or if it's a reference/identifier to a separate name variant object.

Reply:
I did want the REST contract to remain abstract about whether it would contain the name or a reference as this was still to be decided. For REST, it doesn't really matter which solution it is.
We've briefly touched this topic last meeting during the demo of the mockups, and can continue the discussion. To avoid any confusion, it does make sense to update the examples with what will eventually be used, but I assume it doesn't have to block starting the REST API implementation

Reply:
@benbosman : I agree, this isn't a blocker, but the examples here should be updated with more realistic examples (once a decision is made).  Otherwise the vagueness may make it difficult to validate the implementation against the REST contract.

All that said, I'm fine with this contract for now.  I just think it will need an update once implementation decisions have been made.